docs/UserGuide.md

```diff
@@ -297,20 +321,26 @@ Edits an entry, where at least **1** of the optional fields description/amount/d
 | **`[c/EDITED_CATEGORY]`**    | Updated [category](#categorising-your-expenses-and-income) that the entry belongs to                 |
 
 #### Edits expenditure with specified fields
+
 - Example: `edit 1 t/e d/Chicken Soup`
     - The expenditure at the 1st position will have its description edited from `Chicken Rice` to `Chicken Soup`.
 - Expected: `Edited Entry: Chicken Soup; Date: 20-01-2022; Amount: 20.00; Tag: Food`
 
 ![EditCommandExpenditure](images/ug/EditCommandExpenditure.png)
 
 #### Edits income with specified fields
+
 - Example: `edit 2 t/i a/150.00 da/22-10-2022`
     - The income at the 2nd position will have its amount and date edited
 - Expected: `Edited Entry: Monthly Allowance; Date: 02-01-2022; Amount: 150.00; Tag: Allowance`
 
 ![EditCommandIncome](images/ug/EditCommandIncome.png)
 
 ### Deleting entries
+
+You were going to watch the latest Kungfu Panda movie, but your friend bailed on you :(. You already logged the entry so
```

Comment:
Should we add an user story for deleting income too?

Reply:
Have added @EmilyOng, do help me see it is okay :D